scripts/buildPackages.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const _ = require('lodash');
+
+const packages = [
+  {
+    filename: 'keyboard.js',
+    content: `module.exports = require('./lib/components/Keyboard').default;\n`
```

Comment:
Why not use 'components: [keyboard]'?

Reply:
Take a look at `./lib/components/Keyboard`
It already contains all the various Keyboard components we have. 
The user will be able to import it like this
```
import {KeyboardTrackingView, KeyboardAwareInsetsView, KeyboardRegistry, ...} from 'react-native-ui-lib/keyobard';
```

Reply:
I see. mmm... I still think it's clearer to use the same object format, seems less random. Your call.